src/Lib.hs

```diff
@@ -119,16 +119,24 @@ processTicket tId = do
 
 processTickets :: App ()
 processTickets = do
-    sortedTicketIds     <- listAndSortTickets
-    _                   <- mapM (processTicket . tiId) sortedTicketIds
+    sortedTicketIds             <- listAndSortTickets
+    sortedUnassignedTicketIds   <- listAndSortUnassignedTickets
+
+    let allTickets = sortedTicketIds <> sortedUnassignedTicketIds
+
+    _                   <- mapM (processTicket . tiId) sortedUnassignedTicketIds
```

Comment:
This is for the current run.

Reply:
Just curious, can we fetch all the ticket with single fetching function (basically we fetch all the "new", "open", "pending", "hold" tickets and we don't care about whether they're assigned or not) or is that not really a good idea?

Reply:
I don't see why we _have to_ do that. We can if the future requirements of the project dictate so.